pkg/apis/servicecatalog/checksum/unversioned/checksum.go

```diff
@@ -44,3 +44,23 @@ func InstanceSpecChecksum(spec servicecatalog.InstanceSpec) string {
 	sum := sha256.Sum256([]byte(specString))
 	return fmt.Sprintf("%x", sum)
 }
+
+// BindingSpecChecksum calculates a checksum of the given BindingSpec based on
+// the following fields:
+//
+// - InstanceRef.Name
+// - Parameters
+// - OSBGUID
+func BindingSpecChecksum(spec servicecatalog.BindingSpec) string {
+	specString := ""
+	specString += fmt.Sprintf("instanceRef: %v\n", spec.InstanceRef.Name)
+
+	if spec.Parameters != nil {
+		specString += fmt.Sprintf("parameters:\n\n%v\n\n", string(spec.Parameters.Raw))
+	}
+
+	specString += fmt.Sprintf("osbGuid: %v\n", spec.OSBGUID)
```

Comment:
The OSBGuid is static and not expected to change.  In fact, I could probably have omitted it from the checksum.  The fields which are important to checksum are the ones that will mean that the instance needs to be updated at the broker.  Overall I was trying to capture those; it's okay to have fields like the OSBGUID which are static.